app/services/organizations/delete.rb

```diff
@@ -15,5 +18,24 @@ def self.call(...)
     private
 
     attr_reader :org
+    attr_accessor :article_ids
+
+    def delete_notifications
```

Comment:
I planned to use `in_batches.delete_all` but after reading the description of this pr #5303 decided to use the `BulkSqlDelete` approach.